Tests/Fuzzers/RegressionTests/run_tests.sh

```diff
@@ -10,7 +10,11 @@ NC='\033[0m'
 
 for sample in $(ls ${SAMPLES}); do
 	echo -n "Running sample $sample..."
-	$BINARY $SAMPLES/$sample &> /dev/null && echo -e "${GREEN}[OK]${NC}" || { FAILED=True && echo -e "${RED}[FAIL]${NC}"; }
+	if [ -z "$1" ]; then
+		$BINARY $SAMPLES/$sample &> /dev/null && echo -e "${GREEN}[OK]${NC}" || { FAILED=True && echo -e "${RED}[FAIL]${NC}"; }
+	else
+		$BINARY $SAMPLES/$sample && echo -e "${GREEN}[OK]${NC}" || { FAILED=True && echo -e "${RED}[FAIL]${NC}"; }
+	fi
```

Comment:
Is this change needed in this PR? 🤔

Reply:
That was useful to see the output when tests were failing, but I can remove it if you want.

Reply:
I think we can remove it, let me know if you think otherwise

Reply:
I have removed it. Anyway I need to update `.cirrus.yml` if I want to debug the output.